### PR TITLE
Initial commits for sustainability and health 101 metrics

### PR DESCRIPTION
**Pull request by emmairwin**
Sustainability and project health 101 metrics model intial commits.
Includes current status of work from https://docs.google.com/document/d/1UOGJGV17Wl3x5tTK45zuPlhn-AFLV3qi-yvN6rKV79Q/edit as well as Microsoft-specific metrics.

### CODE REVIEW

**Comment by germonprez** [comment]
Thanks @emmairwin :) 

Does the structure you are proposing look like 

Focus Area: Health-101
Metric Model: Sustainability 
Metric Model: Contribution
Metric Model: Discovery
Metric Model: Project-Standards
Metric Model: Responsiveness
Metric Model: Usage 

Focus Area: Sustainability 
Metric Model: Burnout-Risk
Metric Model: Funding
Metric Model: Safety 
Metric Model: Security

**Comment by emmairwin** [comment]
I think so yes

File: focus-areas/health-101/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Sustainability
+
+Goal: We have a set of light-weight metrics and methods required to evaluate the very basic health of an open source project.
+
+# Some Potential User Stories
+
+
+* As a potential consumer of an open source project I want to evaluate the health of engagement around that project before ingesting/using.
+* As someone contributing to open source, I want to assess the health of the project before I invest amy of my time there.
+* As a maintainer of a project, I want to make sure that I have covered all of the basics to ensure my project is setup for success, and track that progress over time
```
Comment (eyehwan) [inline focus-areas/health-101/README.md:10]: Maybe add one more role: Open source foundations to support projects and communities by evaluating its health, to decide in which level support needed, like financial resources etc.

File: focus-areas/health-101/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Sustainability
+
+Goal: We have a set of light-weight metrics and methods required to evaluate the very basic health of an open source project.
+
+# Some Potential User Stories
+
+
+* As a potential consumer of an open source project I want to evaluate the health of engagement around that project before ingesting/using.
```
Comment (eyehwan) [inline focus-areas/health-101/README.md:8]: should we give them a explicit roles, like OSPOs, Corporations, individual users etc...

File: focus-areas/health-101/discovery.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Discovery
+
+
+## Metrics in this model
+This have yet to be merged with CHAOSS metrics
+
```
Comment (eyehwan) [inline focus-areas/health-101/discovery.md:6]: Project popularity

**Comment by emmairwin** [comment]
Thanks @eyehwan for great feedback, and additions, I will review probably a bit later today or Monday. Much appreciated.

**Comment by emmairwin** [comment]
Closing this for now deferring to https://docs.google.com/document/d/1UOGJGV17Wl3x5tTK45zuPlhn-AFLV3qi-yvN6rKV79Q/edit still underway

**Pull request closed without merging**